src/modules/microdds_client/dds_topics.yaml

```diff
@@ -0,0 +1,96 @@
+#####
+#
+# This file maps all the topics that are to be used on the micro DDS client.
```

Comment:
Do you have an estimate for how much bandwidth these topics produce in total on your average quadrotor platform?

It looks like just the `sensors_combined`, `vehicle_attitude`, `vehicle_local_position`, and `vehicle_global_position` alone might have the capacity to take up 500K-700Kbits/s, so without rate limiting/filtering, I'm wondering where we stand with this default message set going over a 1MBaud serial interface.

Reply:
Not really, it all goes back to things like IMU_GYRO_RATEMAX (and the actual IMU), IMU_INTEG_RATE, etc, so the the range in output data rate can be enormous for certain topics. 

In general though I'm going to take a closer look at optimizing some of these higher rate topics, being a bit more careful with the size and maybe do more to reduce the publication rate when there's little to no change.